### PR TITLE
[iCloud] Adding system low battery channel and french localization

### PR DESCRIPTION
**Pull request by clinique**
Signed-off-by: Gaël L'hopital <gael@lhopital.org>

### CODE REVIEW

**Comment by martinvw** [comment]
@ghys / @lolodomo can you maybe review the translations?

File: addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Binding
+icloud.binding.name=Extension iCloud
+icloud.binding.description=Apple iCloud est utilis� pour obtenir des donn�es telles que le niveau de la batterie ou la localisation d'un ou plusieurs appareils Apple connect�s � un compte iCloud. Les mises � jour sont rapides et pr�cises sans impact notable sur la batterie.
+
+# Account Thing
+icloud.account-thing.label=Compte iCloud
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties]: GitHub renders this like cr*p I assume you tested it and it works fine?

File: addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+icloud.account-thing.description=Le 'Thing' compte, plus pr�cis�ment Bridge vers le compte, repr�sente un compte Apple iCloud. Vous pouvez en cr�er plusieurs si vous disposez de plusieurs comptes iCloud.
```
Comment (ghys) [inline addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties]: _'Thing' compte_ feels weird, I didn't understand at first.
I'd suggest:

icloud.account-thing.description=Ce 'Thing', plus pr�cis�ment un Bridge, repr�sente un compte Apple iCloud. Vous pouvez en cr�er plusieurs si vous disposez de plusieurs comptes iCloud.

File: addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+icloud.account-thing.parameter.password.label=Mot de Passe
+icloud.account-thing.parameter.password.description=Mot de passe Apple iCloud pour l'Apple Id
+icloud.account-thing.parameter.refresh.label=Rafra�chir
+icloud.account-thing.parameter.refresh.description=P�riode de rafra�chissement en minutes.
+
+icloud.account-thing.property.owner=Propri�taire
+
+# Device Thing
+icloud.device-thing.label=Equipement iCloud
+icloud.device-thing.description=Le 'Thing' �quipement represente un appareil Apple, tel par exemple qu'un t�l�phone. Il n�cessite d'�tre connect� � un 'Thing' compte pour �tre mis � jour. Plusieurs �quipement peuvent �tre connect�s � un m�me compte.
```
Comment (ghys) [inline addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties]: icloud.device-thing.description=Le 'Thing' �quipement represente un appareil Apple, tel qu'un t�l�phone par exemple. Il n�cessite d'�tre connect� � un 'Thing' compte pour �tre mis � jour. Plusieurs �quipements peuvent �tre connect�s � un m�me compte.

File: addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties (hunk 1 above)
Comment (ghys) [reply addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties]: Properties files can only be ISO-8859-1 per spec AFAIK, that's probably the reason.

**Comment by martinvw** [comment]
@clinique sorry, it seems we merged a conflicting change yesterday, it would be great if could rebase. Sorry for the inconvenience.

@ghys can you validate/approve?

File: addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+icloud.account-thing.parameter.apple-id.label=Apple Id
+icloud.account-thing.parameter.apple-id.description=Apple Id (e-mail) permettant l'acc�s aux informations iCloud.
+icloud.account-thing.parameter.password.label=Mot de Passe
+icloud.account-thing.parameter.password.description=Mot de passe Apple iCloud pour l'Apple Id
+icloud.account-thing.parameter.refresh.label=Rafra�chir
+icloud.account-thing.parameter.refresh.description=P�riode de rafra�chissement en minutes.
+icloud.account-thing.property.owner=Propri�taire
+# Device Thing
+icloud.device-thing.label=Equipement iCloud
+icloud.device-thing.description=Ce 'Thing' repr�sente un appareil Apple, tel par exemple un t�l�phone. Il doit �tre connect� � un 'Thing' Compte pour �tre mis � jour. Plusieurs �quipement peuvent �tre rattach�s � un m�me compte.
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties]: Did you check whether the ' ends up correct on the screen, normally quotes have to be escaped with other quotes in property files, as also shown by the failed rendering.

https://www.mscharhag.com/java/resource-bundle-single-quote-escaping

Comment (martinvw) [reply addons/binding/org.openhab.binding.icloud/ESH-INF/i18n/iCloud_fr.properties]: Just create a new PR if this really broke it, I will merge now :-)

**Comment by martinvw** [comment]
@clinique thanks!

**Comment by martinvw** [comment]
@clinique Some questions where asked in the community forum:

> Hi @martinvw
could you please share an information why the altitude was added?
The value is always zero and doesn´t change at all…
The PR only mentions the lowbattery channel and some french descriptions but not a change at the location.

> Do we really need an altitude when the value is empty and is it possible to remove this value?

> kind regards
Michael

It would be great if you could give us some background about it 😊

**Comment by clinique** [comment]
@martinvw : I just noticed that iCloud API made this information available, and as LocationType is able to handle it, I did add it.

**Comment by martinvw** [comment]
Thanks for your quick reply! Are you btw also a member on the community forum, I could not find you there to mention you :-)

**Comment by clinique** [comment]
Yes, of course I'm ! But under glhopital pseudo

**Pull request merged**